blynklib_mp.py

```diff
@@ -70,6 +70,7 @@ def _pack_msg(self, msg_type, *args, **kwargs):
 
     def parse_response(self, rsp_data, msg_buffer):
         msg_args = []
+        msg_tail = b''
         try:
             msg_type, msg_id, h_data = struct.unpack('!BHH', rsp_data[:self.MSG_HEAD_LEN])
```

Comment:
In general - I am surprised that we have multiple messages in response
For me this is something new ))
How frequently you get multiple messages ? Just interesting

But Ok ...

I suggest calculate before this line  msg_len
By default msg_len=const(0)
Then we get it as len of header+msg_body
There will be no tail - just msg_len that 
not depend on msg_type

Reply:
> How frequently you get multiple messages ? Just interesting
I get it always when have two or more components "Value Display" with reading rate in seconds or quick tap by push button.